Code/Engine/ECS/Components/EditorControllerComponent.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+namespace sad::ecs
+{
+	struct EditorControllerComponent {};
+}
```

Comment:
I remember I mentioned separating components in files 😅 but now I'm wondering if it makes sense to combine the editor and player controller components in one file? I'm cool with joining components in the same `.h` that are manipulated by the same ECS system - just curious what you think.

Also, would probably good to add an `@brief` just explaining what the component does 😄